specification/applicationinsights/resource-manager/readme.md

```diff
@@ -28,7 +28,7 @@ These are the global settings for the ApplicationInsights API.
 title: ApplicationInsightsManagementClient
 description: Composite Swagger for Application Insights Management Client
 openapi-type: arm
-tag: package-preview-2020-10
+tag: package-preview-2021-03
```

Comment:
If you want to use tag `package-preview-2021-03`, you should create a tag `package-preview-2021-03` which inherits from `package-preview-2020-10`. Inherit means to copy all input-files from `package-preview-2020-10`.

Reply:
I'll revert the change to `package-preview-2020-10`.

Reply:
OK. BTW the tag `package-preview-2020-10` is invalid. I will change it to `package-2020-03` later.